CONTRIBUTING.md

```diff
@@ -88,202 +74,54 @@ This is made easier if you use [Chocolatey](https://chocolatey.org), a package m
-
-### Notes and callouts
-
-Notes and callouts can be specified using the blockquote syntax. The converter will look at the first following word to see if it is bold. If so, it will apply that as a callout style. For example:
-
-    > **NOTE** This is a note
-
-Will be displayed as a callout, styled as a "note". The other styles available for callouts are "note", "warning", "tip" and "todo".
-
-### Images
-
-Images can be included by adding the file directly to the repository and adding a link to the image like so:
-
-    ![Alt text](path-to-img.png)
-
-Please, downscale screenshots made at high resolution.
-
-    ![Alt text](path-to-img.png){:width="42px"}
-
-## A word on submodules
-
-The `sdkdocs-template` repo is added as a submodule, and it also contains a submodule to the private `webhelp-template` repo. The `sdkdocs-template` repo contains build time scripts and compiled and minified JS and CSS that allow the site to run. The private `webhelp-template` repo contains the code to build the JS and CSS. It is currently closed source, but the plan is to make it open source at some point, in which case, it is likely the two repos will be merged.
-
-After cloning, a submodule needs to be initialised and updated:
-
-    ```
-    git submodule init
-    git submodule update
-    ```
-
-This will create a `.gitmodules` file, register a submodule in the `sdkdocs-template` folder, and check out the files. (Note that when a repo is added as a submodule, it doesn't get a `.git` folder, but instead gets a `.git` file that points to the location of the `.git` folder.
-
-A submodule can be updated using normal git commands such as `git pull`. It can be switched to a different branch using `git checkout`, and any changes to the currently checked out revision need to be committed back into the main repo, as normal git commands. It is initially cloned at a specific revision, and not as part of a branch.update
-
-If changes are made to the submodule, they should be made on a branch to a clone, and a Pull Request sent. Changes can be made and committed, and the hosting repo will need to commit a pointer to the current version of the submodule.
+If changes are made to the submodule, they should be made on a branch to a clone, and a Pull Request sent. 
+Changes can be made and committed, and the hosting repository will need to commit a pointer to the current version of the submodule.
 
 If there are any problems with the `sdkdocs-template`, please [raise an issue](https://github.com/JetBrains/sdkdocs-template/issues).
 
+## Creating IntelliJ Platform SDK Content
+Content contributions to the IntelliJ Platform SDK are welcome.
+Please download or clone the open source SDK project from [GitHub](https://github.com/JetBrains/intellij-sdk-docs), make additions or changes, and submit a pull request.
+Before creating or altering content, please consult these guides:
+* [SDK Code Sample Guidelines](intro/sdk_code_guidelines.md). 
```

Comment:
order of Code vs Docs links is swapped with menu

Reply:
I'll update the CONTRIBUTING doc to match the _SUMMARY menu.